react/src/base/inputs/SprkDatePicker/SprkDatePicker.js

```diff
@@ -0,0 +1,185 @@
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import TinyDatePicker from 'tiny-date-picker';
+import assign from 'lodash/assign';
+import uniqueId from 'lodash/uniqueId';
+
+class SprkDatePicker extends Component {
+  constructor(props) {
+    const { value } = props;
+    super(props).datePickerInputRef = React.createRef();
+    this.tdpConfig = {
```

Comment:
Can this be in the default? If we're setting it here can we remove it from the default props?

Reply:
@guern1kn  were you able to determine that?

Reply:
I tried moving this around and it breaks everything. With the timing, I feel like we should just leave it as is for right now. If we want to research it we can make another issue for it.

Reply:
ok sounds good